build-aux/bootstrap.sh

```diff
@@ -32,6 +32,7 @@ tools/gn/bootstrap/bootstrap.py -v --no-clean --gn-gen-args='
     rtc_link_pipewire=true
     rtc_pipewire_version="0.3"
     enable_hangout_services_extension=true
+    disable_fieldtrial_testing_config=true
```

Comment:
Could you explain what the impact is of this change, and why it is desirable, in the commit message?